content/100.figure-table-legends.md

```diff
@@ -90,6 +90,23 @@ B. Example heatmap as shown in the cell type summary report comparing annotation
 Heatmap cells are colored by the Jaccard similarity index.
 A value of 1 means that there is complete overlap between which cells are annotated with the two labels being compared, and a value of 0 means that there is no overlap between which cells are annotated with the two labels being compared.
 The heatmap shown is from library `SCPCL000498` [@doi:10.1016/j.devcel.2022.04.003].
+<br><br>
+
+<!-- Figure 5 -->
+![**Consensus cell type annotations in Brain and CNS tumors.**](https://raw.githubusercontent.com/AlexsLemonade/scpca-paper-figures/main/figures/compiled_figures/pngs/figure_5.png?sanitize=true){#fig:fig5 tag="5" width="7in"}
+
+A. Dot plot of marker gene expression across broad consensus cell type annotation groups, specifically considering cells across libraries of brain and central nervous system (CNS) tumors.
+In each panel, the total number of cells in each cell type annotation group across libraries considered is shown in parentheses along the y-axis.
+The x-axis displays marker genes for each cell type annotation group.
+Dots are colored by mean gene expression across libraries and sized proportionally to the percent of libraries they are observed in.
+
+B. Barplot showing the percentage of each broad consensus cell type annotation across libraries of brain and CNS tumors, separated into high-grade (left panel) and low-grade (right panel) glioma diagnoses.
+
+C. Barplot showing the percentages of different consensus cell type immune cells across libraries of brain and CNS tumors, separated into high-grade (left panel) and low-grade (right panel) glioma diagnoses.
```

Comment:
```suggestion
C. Barplot showing all consensus cell types classified as immune cells across libraries of brain and CNS tumors, separated into high-grade (left panel) and low-grade (right panel) glioma diagnoses.
The percentage shown corresponds to the percentage of immune cells classified as the indicated consensus cell type.
```
Just being a little bit more specific about how percentages are determined. Maybe this helps?